migrations/versions/17de7a1bad16_.py

```diff
@@ -0,0 +1,327 @@
+"""empty message
+
+Revision ID: 17de7a1bad16
+Revises: bbf13d065442
+Create Date: 2023-04-21 00:18:06.044764
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '17de7a1bad16'
+down_revision = 'bbf13d065442'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # op.drop_table('bitcoin_prices')
+    # op.drop_table('tmp_import')
+    # op.drop_table('localeth_event_topics')
+    # op.drop_table('localeth_offers')
+    # op.drop_table('localeth_event_logs')
+    # op.drop_table('telegram_messages')
+    # op.drop_table('tmp_identity')
+    # op.drop_table('purse_item')
+    # op.drop_table('purse_offer')
+    # op.drop_table('telegram_chats')
+    # op.drop_table('purse_offer_items')
+    # op.drop_table('telegram_message_hides')
+    # op.drop_table('tmp_eth_contact')
+    # op.drop_table('localeth_payment_methods')
+    # op.drop_table('telegram_user_bans')
+    # op.drop_index('amount', table_name='paxful')
+    # op.drop_index('payment_method_currency', table_name='paxful')
+    # op.drop_index('payment_method_currency_type', table_name='paxful')
+    # op.drop_index('trade_dt', table_name='paxful')
+    # op.drop_table('paxful')
+    # op.drop_table('telegram_users')
+    # op.drop_index('ix_social_stat_name', table_name='social_stat_hourly')
+    # op.drop_index('subscribed_count_ndx', table_name='social_stat_hourly')
+    # op.drop_index('timestamp_and_count', table_name='social_stat_hourly')
+    # op.drop_index('timestamp_ndx', table_name='social_stat_hourly')
+    # op.drop_table('social_stat_hourly')
+    # op.drop_index('ix_circulating_supply_snapshot_date', table_name='circulating_ogn_supply')
+    op.add_column('email_list', sa.Column('source', sa.String(length=255), nullable=True))
+    op.create_index(op.f('ix_email_list_source'), 'email_list', ['source'], unique=False)
+    # op.drop_index('ix_tokens', table_name='eth_contact')
+    # op.drop_index('ix_message_log_email_and_msg_purpose', table_name='message_log')
+    # op.drop_index('name_index', table_name='social_stat')
+    # op.drop_index('subscribed_count_index', table_name='social_stat')
+    # op.drop_index('timestamp_and_name_index', table_name='social_stat')
+    # op.drop_index('timestamp_index', table_name='social_stat')
+    # op.create_index(op.f('ix_social_stat_name'), 'social_stat', ['name'], unique=False)
+    # op.drop_index('ix_token_info_created_at', table_name='token_info')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index('ix_token_info_created_at', 'token_info', ['created_at'], unique=False)
```

Comment:
If these are commented out during the upgrade, let's not "undo" them during downgrade